#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_always_install_polytable.dpatch by Arjan Oosting <arjan@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: make sure we alway include polytable in the Debian package.

@DPATCH@
Index: b/config.mk.in
===================================================================
--- a/config.mk.in
+++ b/config.mk.in
@@ -42,6 +42,6 @@ UNIQ            = @UNIQ@
 
 polydir         = @texmf@/tex/latex/polytable
 texdir          = @texmf@/tex/latex/lhs2tex
-INSTALL_POLYTABLE = @POLYTABLE_INSTALL@
+INSTALL_POLYTABLE = yes
 MKTEXLSR        = @MKTEXLSR@
 
